lib/ddtrace/environment.rb

```diff
@@ -6,7 +6,12 @@ module Environment
     # Defines helper methods for environment
     module Helpers
       def env_to_bool(var, default = nil)
-        ENV.key?(var) ? ENV[var].to_s.strip.downcase == 'true' : default
+        if var.is_a?(Array)
```

Comment:
could do:

```ruby
var = [var] unless var.is_a?(Array)
env_match = var.find { |env_var| ENV.key?(env_var) }
env_match ? ENV[env_match].to_s.strip.downcase == 'true' : default
```

or:

```ruby
if var.is_a?(Array)
  env_match = var.find { |env_var| ENV.key?(env_var) }
else
  env_match = ENV.key?(var)
end

env_match ? ENV[env_match].to_s.strip.downcase == 'true' : default
```

Either could simplify/DRY this a little bit

Reply:
(just a suggestion, not a requirement)

Reply:
updated to be a bit dry'er